ntc_templates/templates/ubiquiti_edgerouter_show_interfaces.textfsm

```diff
@@ -0,0 +1,15 @@
+Value Filldown INTERFACE (\w+)
+Value IP (\S+)
+Value Filldown STATE (\w)
+Value Filldown LINK_STATUS (\w)
+Value Filldown DESCRIPTION ([\w\d\-\s]+)
+
+Start
+  ^[-\s]+ -> Interfaces
+ 
+Interfaces
+  ^${INTERFACE}\s+${IP}\s+(${STATE}/${LINK_STATUS})(\s+${DESCRIPTION})?\s*$$ -> Record
+  ^\s+${IP}\s*$$ -> Record
```

Comment:
I think it makes more sense to make this a list instead of using a filldown and duplicating the interfaces

Reply:
Indeed. I just modified the template by adding the notion of List. Thanks you.

Reply:
Thanks